### PR TITLE
enable allow any for outbound traffic demo profile

### PR DESCRIPTION
**Pull request by linsun**
for demo profile only, per discussion in https://github.com/istio/istio/issues/11739

### CODE REVIEW

**Comment by linsun** [comment]
@costinm since we are documenting all breaking changes from 1.0->1.1, should we introduce this change for the regular values.yaml too?

File: install/kubernetes/helm/istio/values-istio-demo.yaml [hunk 1]
```diff
@@ -6,18 +6,17 @@ global:
...
+  
+  outboundTrafficPolicy:
+    mode: ALLOW_ANY  
 
 pilot:
   traceSampling: 100.0
 
 mixer:
-  policy:
-    enabled: true
```
Comment (rshriram) [inline install/kubernetes/helm/istio/values-istio-demo.yaml]: this as well? our examples in docs will break

Comment (linsun) [reply install/kubernetes/helm/istio/values-istio-demo.yaml]: This is just a duplicate from the default profile.  So no change in behavior

**Comment by ymesika** [comment]
cc @wenchenglu

**Pull request merged**